.github/workflows/ci.yml

```diff
@@ -1,5 +1,5 @@
 name: CI
-on: [push]
+on: [pull_request]
```

Comment:
Didn't consider that 🤔 
But this means we'll have to narrow down this event to ignore stuff we don't care about, right? https://help.github.com/en/articles/events-that-trigger-workflows#pull-request-event-pull_request

Reply:
I assume this has to be opened or ready_for_review